packages/ui/src/components/molecules/JoystickInteraction/Elements.tsx

```diff
@@ -45,12 +46,7 @@ export const Text: React.FC<{ text: string } & DirectionProps> = props => {
 export const JoystickArrow: React.FC<JoystickArrowProps> = props => {
   const { type, state = 'unselected' } = props;
   return (
-    <Image
-      src={typeMap[state]}
-      alt="Arrow"
-      position="absolute"
-      {...directionMap[type]}
-    />
+    <JoystickArrowIcon fill={arrowColorMap[state]} {...directionMap[type]} />
```

Comment:
![image](https://github.com/Cypherock/cypherock-cysync/assets/107414753/c6a05a30-6505-41da-ab6b-93111bc9a6cb)

This icons doesn't seem to be in their positions

Reply:
ah i missed the position parameter my bad